packages/documents/src/pdf/mock-api.ts

```diff
@@ -152,9 +152,10 @@ export const parseAlabamaNameChangeForm = (): ParsedPdf => {
         };
       }
     }
-    if (fieldsetElements.length === 1) {
-      rootSequence.push(fieldsetElements[0]);
-    } else if (fieldsetElements.length > 1) {
+    // if (fieldsetElements.length === 1) {
+    //   rootSequence.push(fieldsetElements[0]);
+    // } else
+    if (fieldsetElements.length > 0) {
       parsedPdf.elements[element.id] = {
```

Comment:
Show more of the form! Produces the final filled pdf just fine. Need to clean up the confirm submission page, but we should skip that for the demo anyway